src/main/java/gigaherz/survivalist/Survivalist.java

```diff
@@ -582,6 +582,7 @@ public void init(FMLInitializationEvent event)
     @Mod.EventHandler
     public void postInit(FMLPostInitializationEvent event)
     {
+    	ConfigManager.instance.parseChoppingAxes();
```

Comment:
This line has a tab character that should have been spaces. It's OK though, I'll fix any indentation problems locally when I do a reformat (I do those often).